src/SAWScript/Crucible/LLVM/X86.hs

```diff
@@ -728,7 +738,7 @@ assumeAllocation ::
   Map MS.AllocIndex Ptr ->
   (MS.AllocIndex, LLVMAllocSpec) {- ^ llvm_alloc statement -} ->
   X86Sim (Map MS.AllocIndex Ptr)
-assumeAllocation env (i, LLVMAllocSpec mut _memTy align sz loc False) = do
+assumeAllocation env (i, LLVMAllocSpec mut _memTy align sz loc False _symInit) = do
```

Comment:
Should we be examining the `symInit` argument here to initialize if necessary?

Reply:
ah, I guess I ignored it in X86. I will fix that